dts/msm8916/msm8916-samsung-gprime.dtsi

```diff
@@ -0,0 +1,18 @@
+lk2nd,keys = <KEY_HOME 109 (GPIO_PULL_UP | GPIO_ACTIVE_LOW)>;
+
+samsung,muic-reset {
+	i2c-gpio-pins = <2 3>;
+	i2c-address = <0x25>;
+};
+
+panel {
+	compatible = "samsung,gprime-panel";
+
```

Comment:
```suggestion

	qcom,mdss_dsi_hx8389c_qhd_video {
		compatible = "samsung,hx8389c-gh9607501a";
	};
```

A possible solution for `qcom,mdss_dsi_hx8389c_qhd_video` (with https://github.com/msm8916-mainline/linux/pull/227/commits/f4391c6e6264ba272306c424470bb83e880582b2)